src/toolbox.js

```diff
@@ -58,15 +58,23 @@ var Toolbox = GObject.registerClass({
         });
         this._realAdd(this._revealer);
 
-        buttonReset.connect('clicked', this._onResetClicked.bind(this));
-        buttonMinimize.connect('clicked', this._onMinimize.bind(this));
+        this._buttonReset.connect('clicked', this._onResetClicked.bind(this));
+        this._buttonMinimize.connect('clicked', this._onMinimize.bind(this));
         this.setBusy(false);
     }
 
     get title() {
         return this._title;
     }
 
+    get resetButton() {
+        return this._buttonReset;
+    }
+
+    get buttonMinimize() {
```

Comment:
I find it confusing that one button is called `resetButton` but the other one is called `buttonMinimize` and not `minimizeButton`.
Either make both start with `button` or not :)